apps/contact-service/src/routes/contacts/importContactsQuotaService.ts

```diff
@@ -0,0 +1,141 @@
+import {Schema} from '@effect/schema'
+import {UnexpectedServerError} from '@vexl-next/domain/src/general/commonErrors'
+import {type HashedPhoneNumber} from '@vexl-next/domain/src/general/HashedPhoneNumber.brand'
+import {UnixMillisecondsE} from '@vexl-next/domain/src/utility/UnixMilliseconds.brand'
+import getNextMidnightOnCurrentDate from '@vexl-next/server-utils/src/getNextMidnightOnCurrentDate'
+import {RedisService} from '@vexl-next/server-utils/src/RedisService'
+import {Context, Effect, Layer} from 'effect'
+import {importContactsQuotaConfig} from '../../configs'
+
+const ImportContactsQuotaRecord = Schema.Int.pipe(
+  Schema.greaterThanOrEqualTo(0)
+)
+
+const createQuotaRecordKey = (hashedPhoneNumber: HashedPhoneNumber): string =>
+  `importContactsQuota:${hashedPhoneNumber}`
+
+const incrementImportContactsQuota =
+  (hashedPhoneNumber: HashedPhoneNumber) =>
+  (numberOfImportedContacts: number) =>
+    Effect.gen(function* (_) {
+      const redis = yield* _(RedisService)
+      const quotaRecordKey = createQuotaRecordKey(hashedPhoneNumber)
+      const oldImportedContactsCount = yield* _(
+        redis.get(ImportContactsQuotaRecord)(quotaRecordKey),
+        Effect.catchTag('RecordDoesNotExistsReddisError', () =>
+          Effect.succeed(0)
+        )
+      )
+
+      const newImportedContactsCount =
+        oldImportedContactsCount + numberOfImportedContacts
+
+      const expiresAt = Schema.decodeSync(UnixMillisecondsE)(
+        getNextMidnightOnCurrentDate()
```

Comment:
move to UnixMilliseconds.brand.ts file and make the function return UnixMilliseconds type